proto/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM golang:1.13.2
+
+ENV PROTOBUF_VERSION="3.10.0"
+ENV PATH="/opt/protoc/bin:${PATH}"
+
+# go mod would stop us from cloning full repos below
+# something like https://github.com/uber/prototool/blob/dev/Dockerfile#L8-L14
```

Comment:
not sure I follow this comment. Looks like you're already using go modules below (`GO111MODULE=on`)